src/rl.py

```diff
@@ -97,6 +97,7 @@ def main():
     parser.add_argument("--evaluate", action="store_true")
     parser.add_argument("--curriculum",action="store_true")
     parser.add_argument("--verifyvval",action="store_true")
+    parser.add_argument("--evaluate-baseline", action="store_true")
```

Comment:
I would move this next to `--evaluate`, and make the two mutually exclusive, like is done with `--proofs` and `--proofs-file` above

Reply:
Then, you should also make it so that if the user passes `--evaluate-baseline` but not `--evaluate`, the evaluation of the baseline still runs (change the if statement on line 386).